### ISSUE TITLE
Admin users with user scope and admin scope permission for auditlog don't see audit log

### ISSUE THREAD

**Issue by berndnicklas** [privacyidea/privacyidea#530]
# Versions

privacyIDEA: trunk
Installation method:
- [x] from source / github
- [ ] in virtualenv
- [x] Debian
- [ ] Ubuntu
- more details:
  OS: Debian Jessie
  Webserver: NGINX / UWSGI
# What did you try to do?

With configuration

Policy "user_support_l1"
Scope: user
Action: {"auditlog": true}
Realm: ["support_l1"]

Policy "admin_support_l1"
Scope: admin
Action: {"auditlog": true}
Realm:  ["staff"]

logged in as admin user into admin realm "support_l1" and viewed audit log. 
# What outcome did you expect?

Can view audit log of realms "support_l1" (defined in user scope policy) and "staff" (defined in admin scope policy).
# What outcome did you experience?

"Audit" item in main menu is visible, but clicking on it brings an empty audit log and the error message 

```
Admin actions are defined, but the action auditlog is not allowed!
```
# Log file

There are no logs for this error on server side.

**Comment by berndnicklas** [comment]
What outcome did you expect?

Ah, better would be:
- Can view **own** audit log entries of realms "support_l1" (defined in user scope policy).
- Can view **all** audit entries of realm "staff" (defined in admin scope policy).

**Comment by cornelinux** [comment]
This references to #541

**Comment by cornelinux** [comment]
This is closed with #663
